products/emberjs.md

```diff
@@ -10,7 +10,7 @@ changelogTemplate: https://github.com/emberjs/ember.js/releases/tag/v__RELEASE_C
 activeSupportColumn: true
 releaseDateColumn: true
 releases:
-  - releaseCycle: "4.x"
```

Comment:
Trying to avoid `.x` in releaseCycles since it shows up in our API. Using `4` makes it easier to consume